src/libstd/keyword_docs.rs

```diff
@@ -1030,9 +1030,85 @@ mod self_upper_keyword {}
 //
 /// A place that is valid for the duration of a program.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// A `static` item is similar to a [`const`] item in that it lives for the
+/// entire duration of the program and need to have its type explicited, with a
+/// `static` lifetime, outliving any other lifetime. Added to that, `static`
+/// items represent a precise memory location.
```

Comment:
I'm not terribly happy with this paragraph, unfortunately :/

- While I see where you're going with this, I would avoid saying "`const`s live for the whole program", as they work quite different from `static` variables.
- "have its type explicited": I think the word is "explicated", but apart from that, I never heard that formulation and "explicitly annotated" would be better IMO. I am not a native speaker though, so if it's a valid formulation, probably ignore this remark. 
- The first sentence is fairly long (as in: has many parts). Nothing super critical, but splitting the sentence doesn't hurt and makes it a touch easier to read, IMO.

Here is an alternative (also of the previous paragraph) of which you can take everything, parts or nothing:

> A `static` item is a value which is valid for the entire duration of your program (a `'static` lifetime).
>
> On the surface, `static` items seem very similar to `const`s: both contain a value, both require type annotations and both can only be initialized with constant functions. However, `static`s are notably different in that they represent a location in memory. That means that you can have references to `static` items or potentially even modify them, making them essentially global variables.

Reply:
*have its type explicited* is, to the best of my knowledge, a valid formulation. I agree it is very heavy and I used it because I couldn't find something better when I wrote it.

You did though, so I used your paragraph. Thanks !